lib/configure.js

```diff
@@ -80,6 +80,29 @@ function configure (gyp, argv, callback) {
     }
   }
 
+  function getModulesVersion(nodeDir) {
+
+    var sources = [
+      path.join(nodeDir, 'include/node/node_version.h'),
+      path.join(nodeDir, 'src/node_version.h'),
+    ]
+
+    for (var i = 0; i < sources.length; ++i) {
+      try {
+        var source = fs.readFileSync(sources[i], { encoding: 'utf8'})
```

Comment:
The try/catch should only be around this statement.  Style nit: space before `}`.